spec/requests/transformation_mappings_spec.rb

```diff
@@ -1,31 +1,46 @@
-describe "Transformation Mappings" do
+RSpec.describe "Transformation Mappings", :v2v do
   let(:source_ems) { FactoryBot.create(:ems_vmware) }
-  let(:dest_ems) { FactoryBot.create(:ems_redhat) }
-  let(:source_cluster) { FactoryBot.create(:ems_cluster, :ext_management_system => source_ems) }
-  let(:destination_cluster) { FactoryBot.create(:ems_cluster, :ext_management_system => dest_ems) }
-
-  let(:source_redhat_host) { FactoryBot.create(:host_redhat, :ems_cluster => source_cluster) }
-  let(:source_storage) { FactoryBot.create(:storage, :hosts => [source_redhat_host]) }
-
-  let(:destination_vmware_host) { FactoryBot.create(:host_vmware, :ems_cluster => destination_cluster) }
-  let(:destination_storage) { FactoryBot.create(:storage, :hosts => [destination_vmware_host]) }
-
-  let(:source_lan) { FactoryBot.create(:lan) }
-  let(:destination_lan) { FactoryBot.create(:lan) }
+  let(:source_clusters) { FactoryBot.create_list(:ems_cluster, 2, :ext_management_system => source_ems) }
+  let(:source_hosts) { FactoryBot.create_list(:host_vmware, 1, :ems_cluster => source_clusters.first) }
+  let(:source_storages) { FactoryBot.create_list(:storage, 2, :hosts => source_hosts) }
+  let(:source_switches) { FactoryBot.create_list(:switch, 1, :host => source_hosts.first) }
```

Comment:
This should be `FactoryBot.create_list(:switch, 1, :hosts => source_hosts)`

Reply:
Updated.